2005-02-23  Alfred M. Szmidt  <ams@gnu.org>

        * Makefile (iso9660fs.static): New target.

Index: Makefile
===================================================================
RCS file: /cvsroot/hurd/hurd/isofs/Makefile,v
retrieving revision 1.6
retrieving revision 1.7
diff -u -r1.6 -r1.7
--- isofs/Makefile	29 Sep 2002 20:58:34 -0000	1.6
+++ isofs/Makefile	23 Feb 2005 00:00:34 -0000	1.7
@@ -1,5 +1,5 @@
 
-#   Copyright (C) 1997, 2000 Free Software Foundation
+#   Copyright (C) 1997, 2000, 2005 Free Software Foundation
 #
 #   This program is free software; you can redistribute it and/or
 #   modify it under the terms of the GNU General Public License as
@@ -27,3 +27,5 @@
 HURDLIBS = diskfs iohelp fshelp store pager threads ports ihash shouldbeinlibc
 
 include ../Makeconf
+
+iso9660fs.static: $(boot-store-types:%=../libstore/libstore_%.a)
